docs/release_notes.md

```diff
@@ -1,5 +1,46 @@
 # Release Notes
 
+## Version 1.4.0
+
+### Core
+
+- Implemented a new solver to make muscles behave more compact: AdnGlue solver.
+- Implemented Soft Constraints.
```

Comment:
If you are referring to soft constraints in AdnGlue maybe we could say "Implemented Glue Constraints" as this is the name we expose for the user.

Reply:
Good idea, done!